tests/integration/testdata/buildcmd/terraform/image_based_lambda_functions_local_backend/main.tf

```diff
@@ -1,3 +1,16 @@
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "2.25.0"
+    }
+  }
+}
+
+provider "docker" {
+  host = "unix:///var/run/docker.sock"
```

Comment:
Why do we need to add docker socket location as well?

Reply:
I don't think we do actually. Let me remove it.